src/renderer/routes/test_sequencer_panel/components/DataTable.tsx

```diff
@@ -205,16 +209,21 @@ export function DataTable() {
       },
     },
 
-    // {
-    //   accessorKey: "isSavedToCloud",
-    //   header: "Saved to Flojoy Cloud",
-    //   enableHiding: false,
-    //   cell: ({ row }) => {
-    //     return row.getValue("isSavedToCloud") ? (
-    //       <Button>OPEN TEST</Button>
-    //     ) : null;
-    //   },
-    // },
+    {
+      accessorFn: (elem) => {
+        return elem.type === "test" ? "isSavedToCloud" : null;
+      },
+      header: "Saved To Cloud",
+      cell: ({ row }) => {
+        const cloudStatus =
```

Comment:
Can be simplified to:

```suggestion
               return row.original.type === "test" ? (
          <div
            className={cn({
              "text-green-500": row.original.isSavedToCloud,
              "text-red-500": !row.original.isSavedToCloud,
            })}
          >
            {row.original.isSavedToCloud ? "Saved" : "Not Saved"}
          </div>
        ) : null;
```